packages/field-base/src/input-field-mixin.js

```diff
@@ -164,29 +148,20 @@ const InputFieldMixinImplementation = (superclass) =>
     }
 
     /**
-     * @param {HTMLElement} node
+     * @param {HTMLInputElement} input
      * @protected
      */
-    _addInputListeners(node) {
-      node.addEventListener('input', this._boundOnInput);
-      node.addEventListener('blur', this._boundOnBlur);
-      node.addEventListener('focus', this._boundOnFocus);
-    }
+    _addInputListeners(input) {
```

Comment:
Since we added a `_removeListeners` in `InputMixin`, it should be implemented here as well.

Reply:
Done.